module/VuFind/src/VuFind/Log/Handler/MailHandler.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * Mail log handler
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Error_Logging
+ * @author   Sambhav Pokharel <sambhav.pokharel@gmail.com>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\Log\Handler;
+
+use Monolog\Handler\MailHandler as MonologMailHandler;
+use Monolog\LogRecord;
+use VuFind\Mailer\Mailer;
+
+use function sprintf;
+
+/**
+ * Custom Mail Handler for VuFind with verbosity support and VuFind mailer integration
+ *
+ * @category VuFind
+ * @package  Error_Logging
+ * @author   Demian Katz <demian.katz@villanova.edu>
```

Comment:
Another case of mismatched author tags. Not a big deal, obviously, but maybe worth adjusting.